apps/meteor/tests/end-to-end/api/25-teams.js

```diff
@@ -1765,11 +1844,18 @@ describe('[Teams]', () => {
 			});
 
 			it('should update team channel to auto-join', async () => {
-				const response = await request.post(api('teams.updateRoom')).set(credentials).send({
-					roomId: createdRoom.body.channel._id,
-					isDefault: true,
-				});
-				expect(response.body).to.have.property('success', true);
+				await request
+					.post(api('teams.updateRoom'))
+					.set(credentials)
+					.send({
+						roomId: createdRoom.body.channel._id,
+						isDefault: true,
+					})
+					.expect(200)
+					.expect((res) => {
+						expect(res.body).to.have.property('success', true);
+						expect(res.body).to.have.nested.property('room.usersCount').and.to.be.equal(2);
+					});
```

Comment:
We don't need to change this assertion/test case, we are already testing this behavior. Let's revert  this